apple-pie/styles/style.css

```diff
@@ -1,6 +1,7 @@
 body {
     margin: 0px;
     width: 600px;
+    font-family: "Proxima Nova Light";
 }
```

Comment:
A suggestion here, if you write 
`margin: 0px auto;` or `margin: auto;`

This would align the content of the body 'automatically' in the center of the browser window. In that case, you could increase width . For regular screen it will be full width and for the very wide screen - it will only fill the middle of the window